src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -62,7 +63,7 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
                   data-test={`${interfaceType}-${iface.name}-open-drawer`}
                 >
                   {iface.name}
-                  <Icon color="black" className="pf-u-mr-sm" />
+                  <Icon className="pf-u-mr-sm pf-u-color-200" />
```

Comment:
I'd better start putting styling into separate css file, as it seems it is increasing here..

Reply:
Done